src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -23,8 +27,26 @@ import App from 'lando/lib/app';
  */
 const DEBUG_KEY = '@automattic/vip:bin:dev-environment-lando';
 const debug = debugLib( DEBUG_KEY );
+const mkdtemp = promisify( fs.mkdtemp );
 
-function getLandoConfig() {
+let landoConfRoot;
+
+/**
+ * @returns {Promise<string>} User configuration root directory (aka userConfRoot in Lando)
+ */
+async function getLandoUserConfigurationRoot() {
+	if ( ! landoConfRoot ) {
+		const tmpDir = os.tmpdir();
+		landoConfRoot = await mkdtemp( `${ tmpDir }${ path.sep }lando` );
```

Comment:
Could we use `path.join` instead of concatenating strings here?

Reply:
Fixed in b8b447e